jetty-distribution/pom.xml

```diff
@@ -358,7 +358,7 @@
               <arguments>
                 <argument>jetty.home=${assembly-directory}</argument>
                 <argument>jetty.base=${assembly-directory}/demo-base</argument>
-                <argument>--add-to-startd=server,continuation,deploy,websocket,ext,resources,client,annotations,cdi2,decorate,jndi,servlets,jsp,jstl,http,https,threadpool</argument>
+                <argument>--add-to-startd=server,continuation,deploy,websocket,ext,resources,client,annotations,cdi-spi,jndi,servlets,jsp,jstl,http,https,threadpool</argument>
```

Comment:
We have demo-base support for CDI? (or Decorators)?

Reply:
Demo base deploys the test-weld-cdi-webapp which needs one CDI integration module.  We have CDITests that tests them all.

Reply:
Does that include the weld jar in the demo-base?
If so, we can't ship it in our jetty-distribution without a CQ.

Reply:
Ah - I had assumed we had one.... but I guess that is why it was not in demo-base before.  I will remove....